File: src/main/kotlin/org/kite9/diagram/common/HintMap.java
package org.kite9.diagram.common;

import java.util.HashMap;
import java.util.Map;

public class HintMap extends HashMap<String, Float>{

	private static final long serialVersionUID = 8622279690962224111L;

	public HintMap() {
		super();
	}

	public HintMap(int initialCapacity, float loadFactor) {
		super(initialCapacity, loadFactor);
	}

	public HintMap(int initialCapacity) {
		super(initialCapacity);
	}

	public HintMap(Map<? extends String, ? extends Float> m) {
		super(m);
	}

}

File: src/main/kotlin/org/kite9/diagram/common/HintMap.kt
package org.kite9.diagram.common

import java.util.HashMap

/**
 * @deprecated
 */
class HintMap : HashMap<String?, Float?> {
    constructor() : super() {}
    constructor(initialCapacity: Int, loadFactor: Float) : super(initialCapacity, loadFactor) {}
    constructor(initialCapacity: Int) : super(initialCapacity) {}
    constructor(m: Map<out String?, Float?>?) : super(m) {}

    companion object {
        private const val serialVersionUID = 8622279690962224111L
    }
}